website/versioned_docs/version-2.0.0-alpha.72/using-plugins.md

```diff
@@ -154,6 +154,30 @@ module.exports = function (context, options) {
 
 The `my-plugin` folder could also be a fully fledged package with it's own package.json and a `src/index.js` file for example
 
+### Functional definition {#functional-definition}
```

Comment:
this should be documented in website/docs, not alpha 72 folder, as it is likely to be merged for alpha 74 when we create that version

Reply:
Not sure why but Netlify reports broken links from this page:

![image](https://user-images.githubusercontent.com/749374/115414447-55e48b00-a1f6-11eb-8fe4-6adfae2b1e50.png)

Can you look at this and try to build the Docusaurus site locally?

Let me know if you need help